libs/dyn/merge/override.go

```diff
@@ -13,11 +13,17 @@ import (
 // For instance, it can disallow changes outside the specific path(s), or update
 // the location of the effective value.
 //
+// Values returned by 'VisitDelete', 'VisitInsert' and 'VisitUpdate' are used as
+// the final value of the node. 'VisitDelete' should return dyn.NilValue to
+// do the delete, or can return 'left' to undo it.
```

Comment:
Oh, we can return a structure describing the decision, otherwise, I don't think there is a way to encode it. But this will be more complex, and I'm not sure it's worth it right now.

Reply:
I agree it's not worth it right now.